<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressBookDialog</class>
 <widget class="QDialog" name="AddressBookDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address Book Properties</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general_tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_name">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_file">
             <property name="text">
              <string>File:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_encryption">
             <property name="text">
              <string>Encryption:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_password">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_password_repeat">
             <property name="text">
              <string>Password (repeat):</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="edit_name">
             <property name="maxLength">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_file">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combo_encryption"/>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_password">
             <property name="maxLength">
              <number>64</number>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_password_repeat">
             <property name="maxLength">
              <number>64</number>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_comment">
           <property name="text">
            <string>Comment:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="edit_comment"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Routers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="button_add_router">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/console.qrc">
             <normaloff>:/img/plus.png</normaloff>:/img/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_modify_router">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/console.qrc">
             <normaloff>:/img/pencil.png</normaloff>:/img/pencil.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_delete_router">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/console.qrc">
             <normaloff>:/img/minus.png</normaloff>:/img/minus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="tree_routers">
         <property name="indentation">
          <number>0</number>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Port</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advanced_tab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Password hashing</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_password_salt">
            <property name="text">
             <string>Size of salt for password hashing:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinbox_password_salt">
            <property name="suffix">
             <string> bytes</string>
            </property>
            <property name="minimum">
             <number>64</number>
            </property>
            <property name="maximum">
             <number>1048576</number>
            </property>
            <property name="value">
             <number>256</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Salt of data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_salt_before">
                <property name="text">
                 <string>Size of salt before the data:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_salt_after">
                <property name="text">
                 <string>Size of salt after the data:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QSpinBox" name="spinbox_salt_before">
                <property name="suffix">
                 <string> bytes</string>
                </property>
                <property name="minimum">
                 <number>64</number>
                </property>
                <property name="maximum">
                 <number>16777216</number>
                </property>
                <property name="value">
                 <number>512</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinbox_salt_after">
                <property name="suffix">
                 <string> bytes</string>
                </property>
                <property name="minimum">
                 <number>64</number>
                </property>
                <property name="maximum">
                 <number>16777216</number>
                </property>
                <property name="value">
                 <number>512</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>74</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/console.qrc"/>
 </resources>
 <connections/>
</ui>
